docs-src/modules/developing/pages/slices.adoc

```diff
@@ -38,7 +38,7 @@ entity_filter = EntityPercentFilter(filter_percentage)
 ----
 
 The example above creates a new `EntityPercentFilter` from the Kaskada
-Compute module with a filtering percentage of 12.34%.
+Slice Filters module with a filtering percentage of 12.34%.
```

Comment:
can it be the `Slicing` module instead?  

slice filters module doesn't roll off the tongue well.